handlers/events_streamer.go

```diff
@@ -57,7 +57,8 @@ func (h *Handler) EventStreamHandler(w http.ResponseWriter, req *http.Request, _
 	// defer close(newAdaptersChan)
 
 	go func() {
-		for mClient := range newAdaptersChan {
+		for loopVar := range newAdaptersChan {
```

Comment:
The change does not make sense to me.

Reply:
```
for mClient := range newAdaptersChan {
      log.Debug("received a new mesh client, listening for events")
      go func(mClient *meshes.MeshClient) {
	      listenForAdapterEvents(req.Context(), mClient, respChan, log)
	      _ = mClient.Close()
}(mClient)
```

Reply:
@leecalcote Any idea?

Reply:
> These codes below make more sense to me
> 
> ```
> for mClient := range newAdaptersChan {
>       log.Debug("received a new mesh client, listening for events")
>       go func(mClient *meshes.MeshClient) {
> 	      listenForAdapterEvents(req.Context(), mClient, respChan, log)
> 	      _ = mClient.Close()
> }(mClient)
> ```

I agree with you, this makes more sense. All I did above was a makeover or can be termed as a cheat to bypass the rule for loop variable.

Reply:
@Aisuko Fixed, Thanks for the suggestions !